src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
OK, so you do the interactive stuff in the
scripts - this is the problem, please don't.
Please do the menu stuff in install.c only.
The script should support --list for install.c
to be able to create the menu properly.

Having freedos on D may still make sense:
1. If user made C unbootable, dosemu will still boot
2. On D freedos can refer to the system-wide location
allowing for an updates.

Maybe for freedos we should support both C: (per-user)
and D: (system-wide) install, and only C: for other doses?